### ISSUE TITLE
getting "Wrong password" when pasting password (vs manually typing it)

### ISSUE THREAD

**Issue by maniqui** [mailvelope/mailvelope#693]
When pasting a password (previously copied from password manager into clipboard), I get a "Wrong password" message, both when attempting to decrypt a message as well as when attempting to sign a message. 

![image](https://user-images.githubusercontent.com/333695/62338223-1ddf6680-b4ae-11e9-9f36-406538cce0f8.png)

Workaround: manually typing the password works as expected. 

Mailvelope: v4.0.0.
Browser: Chromium v76.0.3809.87 / Debian Sid

Other coworkers running Mailvelope on Linux and on macOS were able to reproduce this issue.

**Comment by katebee** [comment]
Same issue here once upgrading to 4.0.0.

Getting a `wrong password` error when pasting. Typing the entire passphrase in does work.
I also tried pasting, deleting the last few characters and re-entering them, however that still fails.

Mailvelope: v4.0.0
Browser: Google Chrome Version 76.0.3809.87 (Official Build) (64-bit)

**Comment by rvandermeulen** [comment]
Same on Firefox Nightly

**Comment by maniqui** [comment]
I've found a workaround by commenting out line **1919** on 
file `~/.config/chromium/Default/Extensions/kajibbejlbohfaggdiogboambcijhkke/4.0.0_0/components/enter-password/passwordDialogRoot.bundle.js`

So, it should look something like this:
```
      value: this.state.password,
      // onPaste: e => this.onInputPaste(e.clipboardData.getData('Text')),
      onChange: e => this.onInputPaste(e.target.value),
```

Then, reload the extension by clicking "Reload Mailvelope" on the Mailvelope extension icon.

Not sure which are the full consequences of commenting out this line, but I got copy-pasted passwords to work again on Mailvelope v4.0.0.

**Comment by panther2** [comment]
Looks like it has been already fixed in the dev branch:
https://github.com/mailvelope/mailvelope/commit/5f7790f923ca8e691bea4605242e0fb23a215228

**Comment by arjonny** [comment]
I was having this issue last night in the chrome extension was making myself mental trying to type out the entire password correctly, repeatedly

**Comment by ayr-ton** [comment]
@arjonny It seems to be a different issue as this one is related to the `onPaste` event.

**Comment by toberndo** [comment]
Fixed with v4.0.1
